tests/e2escenarios/e2e_test.go

```diff
@@ -259,6 +259,230 @@ var _ = Describe("E2E Test", func() {
 			stdout = helper.Cmd("odo", "deploy").AddEnv("PODMAN_CMD=echo").ShouldPass().Out()
 			Expect(stdout).To(ContainSubstring("Your Devfile has been successfully deployed"))
 
+			command = []string{"odo", "delete", "component"}
+			_, err = helper.RunInteractive(command, nil, func(ctx helper.InteractiveContext) {
+				helper.ExpectString(ctx, "Are you sure you want to delete \""+componentName+"\" and all its resources?")
+				helper.SendLine(ctx, "y")
+				helper.ExpectString(ctx, "successfully deleted")
+			})
+			Expect(err).To(BeNil())
+			Eventually(string(commonVar.CliRunner.Run(getDeployArgs...).Out.Contents()), 60, 3).ShouldNot(ContainSubstring(deploymentName))
+			Eventually(string(commonVar.CliRunner.Run(getSVCArgs...).Out.Contents()), 60, 3).ShouldNot(ContainSubstring(serviceName))
+		})
+	})
+	Context("starting with non-empty Directory add Binding", func() {
+		componentName := helper.RandString(6)
+		var _ = BeforeEach(func() {
+			commonVar.CliRunner.EnsureOperatorIsInstalled("service-binding-operator")
+			commonVar.CliRunner.EnsureOperatorIsInstalled("cloud-native-postgresql")
+			Eventually(func() string {
+				out, _ := commonVar.CliRunner.GetBindableKinds()
+				return out
+			}, 120, 3).Should(ContainSubstring("Cluster"))
+			helper.Chdir(commonVar.Context)
+			helper.CopyExample(filepath.Join("source", "devfiles", "nodejs", "project"), commonVar.Context)
```

Comment:
To have a more realistic end-to-end test case, maybe we could consider having a project that actually consumes the binding information?